panko/requirements.txt

```diff
@@ -0,0 +1 @@
+ pydantic_ai==0.1.3
```

Comment:
actually used corpora ai instead but .. meh, this isn't referenced anyway .. hrm .. could be confusing though to come across it .. but .. tombstone for next iteration ... I'm still open to maybe using pydantic ai but .... corpora supports XAI and pydantic currently doesn't 🤔